tools-local/unix/tool-helper.sh

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env bash
+
+# Provides helper functions.
+
+# Gets name of the operating system.
+# Exit 1 if unable to get operating system name.
+get_os_name()
+{
+    osName="$(uname -s)"
+
+    if [ $? -ne 0 ] || [ -z "$osName" ]; then
+        echo "Unable to determine the name of the operating system."
+        exit 1
+    fi
+
+    if echo "$osName" | grep -iqF "Darwin"; then
+        osName="OSX"
+    else
+        osName="Linux"
+    fi
+
+    echo "$osName"
+}
+
+# Eval .toolversions file.
+# TODO: 
```

Comment:
I think this is better tracked with an issue.  TODO comments in code get obsolete quickly.  If we change direction, they require code changes to update.